packages/web3-eth-accounts/src/account.ts

```diff
@@ -0,0 +1,63 @@
+import { utils, getPublicKey } from 'ethereum-cryptography/secp256k1';
+import { toChecksumAddress, bytesToHex, sha3Raw } from 'web3-utils';
+import { PrivateKeyError } from './errors';
+// Will be added later
+export const encrypt = (): boolean => true;
+
+// Will be added later
+export const sign = (): boolean => true;
+
+// Will be added later
+export const signTransaction = (): boolean => true;
+
+/**
+ * Get address from private key
+ */
+export const privateKeyToAccount = (
```

Comment:
1.x doc also have optional `ignoreLengthCheck` boolean but I dnt see any ignoring usage scenario for that, as PK is 64 random hex characters or 32 random bytes, that must be checked.

Reply:
Yes It isn't a useful parameter but it is a breaking change, i'll document that.